src/main/webapp/popup.js

```diff
@@ -13,3 +15,31 @@ function openEventPopup() {
 function closeEventPopup() {
     document.getElementById('add-event').style.display = 'none';
 }
+
+/**
+ * Creates a new Event object and add it to the event list
+ */
+function createNewEvent() {
+    const title = document.getElementById('title').value;
+    const location = document.getElementById('location').value;
+    const startingTime = document.getElementById('starting-time').value;
+    const endingTime = document.getElementById('ending-time').value;
+
+    // Proceed only if all the fields are completed
+    if (!title || !location || !startingTime || !endingTime) {
+        return;
```

Comment:
Don't do it silently, make it visible to user somehow. Alert/red box for missing field/text above button...

Reply:
It is already done by the html elemetns. If a field is empty they will receive a notification

Reply:
This is just so I don't add an event in my LocalSotrage that has empty fields. The user gets notified with or without this (because of the "<input required>" format in HTML)